dashboard/app/models/ability.rb

```diff
@@ -96,6 +96,7 @@ def initialize(user)
       can :create, UserLevel, user_id: user.id
       can :update, UserLevel, user_id: user.id
       can :create, StudentWorkEvaluation, user_id: user.id
+      can :get_student_work_evaluations, StudentWorkEvaluation, user_id: user.id
```

Comment:
🤔 I wonder if only the student and their teacher should be able to get the student's student work evaluations?

Reply:
Good catch!  I am actually wondering for right now if just the teacher should be able to get the evaluations?  This is what I implemented in the changes...